s3api/cmd/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/micro/go-web"
+	"github.com/opensds/multi-cloud/s3api/pkg/filters/context"
+	"github.com/opensds/multi-cloud/s3api/pkg/filters/logging"
+	"github.com/opensds/multi-cloud/s3api/pkg/filters/signature/signer"
+	"github.com/opensds/multi-cloud/s3api/pkg/s3"
+	"github.com/opensds/multi-cloud/s3api/pkg/utils/obs"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	serviceName = "gelatos3"
+)
+
+func main() {
+	webService := web.NewService(
+		web.Name(serviceName),
+		web.Version("v0.1.0"),
+	)
+	webService.Init()
+
+	obs.InitLogs()
+	wc := restful.NewContainer()
+
+	s3ws := new(restful.WebService)
+	s3ws.Path("/")
+	s3ws.Doc("OpenSDS Multi-Cloud API")
+	s3ws.Consumes(restful.MIME_XML)
+	s3ws.Produces(restful.MIME_XML)
```

Comment:
I dont know the  the reason for their existence of "s3ws.Consumes(restful.MIME_XML) and s3ws.Produces(restful.MIME_XML)",  and when I test s3 compatibility with this pr,  I found "s3ws.Consumes(restful.MIME_XML)" lead to error

Reply:
Could you please share what kind of error?

Reply:
"Consumes specifies that this WebService can consume one or more MIME types. Http requests must have one of these values set for the Content-Type header."
Which kind of content type you used when error occured?

Reply:
As this part is the same as before, and this PR is used to fix the issue that URL and authentication are not compatible aws s3. So can check and fix it in the following PR?

Reply:
http status code 415 was returned when I test s3 compatibility,  I commented "s3ws.Consumes(restful.MIME_XML)"  and the test gone on.  
I agree you that we check the issue in following PR.